ranger/ext/rifle.py

```diff
@@ -113,6 +113,11 @@ def squash_flags(flags):
     return ''.join(f for f in flags if f not in exclude)
 
 
+def shell_quote(string):
```

Comment:
I'm sure shlex has a function for this, no?

Reply:
Not in the `python2` version. I copied this from `shell_escape.py`.

Reply:
Let's take [this approach](https://stackoverflow.com/questions/26790916/python-3-backward-compatability-shlex-quote-vs-pipes-quote).